racket/collects/racket/hash.rkt

```diff
@@ -63,6 +63,24 @@
                     (combine/key k v (hash-ref hm k))))
         res)))
 
+(define (hash-filter-values ht pred)
+  (let* ([mutable (not (immutable? ht))]
```

Comment:
Nit: use `define` instead of `let*`.

Reply:
Also, it's not use mutable vs immutable. There're also weak hash table and ephemeron hash table.